test/mocks/MockChallenge.sol

```diff
@@ -31,16 +36,20 @@ contract MockChallenge is IChallenge {
     return mock_passed;
   }
 
-  function reward() public view returns (uint) {
-    return mock_reward;
+  function winnerReward() public view returns (uint) {
+    return mock_winnerReward;
   }
 
   function challenger() view returns (address) {
     return mock_challenger;
   }
 
+  function fundsRequired() view returns (uint) {
+    return mock_fundsRequired;
+  }
+
   function approveRewardTransfer (ERC20 token, address registry) public {
```

Comment:
@emaG3m I think this can be removed now, as well as the spot where this is called in the tests, since there's no longer a reward transfer